plugin.yaml

```diff
@@ -1,9 +1,9 @@
 name: "unittest"
-version: "0.1.5-rancher1"
+version: "0.1.7-rancher1"
 usage: "unittest for helm charts"
 description: "Unit test for helm chart in YAML with ease to keep your chart functional and robust."
 ignoreFlags: false
 command: "$HELM_PLUGIN_DIR/untt"
 hooks:
-  install: "$HELM_PLUGIN_DIR/install-binary.sh"
-  update: "$HELM_PLUGIN_DIR/install-binary.sh"
+  install: "cd $HELM_PLUGIN_DIR; $HELM_PLUGIN_DIR/install-binary.sh"
```

Comment:
`cd $HELM_PLUGIN_DIR; ./install-binary.sh` ?

Reply:
could also do `cd $HELM_PLUGIN_DIR && ./install-binary.sh` in case for some reason that cd fails? dunno.. seems wrong the way it's written

Reply:
I reverted it back - I agree that a full path seems like the best way to approach it vs a cd.